string name
string robot_model
uint8 xml_type
uint8 URDF            = 0  # robot_model contains XML string for an URDF model
uint8 GAZEBO_XML      = 1  # robot_model contains XML string for an URDF model
uint8 URDF_PARAM_NAME = 2  # robot_model contains parameter name containing URDF XML
Pose initial_pose
string model_name
string namespace
---
bool success
string maybe_comment
